src/IPC/PipeRx.cc

```diff
@@ -1,22 +1,52 @@
-#include <iostream>
 #include "IPC.h"
 
+#include <iostream>
+#include <fcntl.h>      /* O_RDONLY */
+#include <unistd.h>     /* read() close() unlink() */
+#include <sys/stat.h>   /* mkfifo() */
+#include <errno.h>  
+#include <string.h>
+
+#define FIFO "./fifoChannel"
+
 void PipeRx::receive(void) {
     this->setupPipeRx();
 
-    this->fileSizeRx();
-
     this->pipeRx();
 }
 
 void PipeRx::setupPipeRx(void) {
-    ;
-}
-
-void PipeRx::fileSizeRx(void) {
-    ;
+    mkfifo(FIFO, 0666);
```

Comment:
for readability purposes I recommend you to replace 0666 by S_IRWXU | S_IRWXG.
Not so important tho.